config/application.properties

```diff
@@ -1,7 +1,11 @@
 #Bridge related settings
 bridge.id=my-bridge
-# uncomment the following line to enable Jaeger tracing, check the documentation how to configure the tracer
+# uncomment one the following lines (bridge.tracing) to enable Jaeger tracing, check the documentation how to configure the tracer
+# OpenTracing support
 #bridge.tracing=jaeger
+# OpenTelemetry support
+#bridge.tracing=jaeger-otel
+#bridge.tracing.service-name=strimzi-kafka-bridge
```

Comment:
Dunno. @ppatierno ? ^

Reply:
So you could customize your spans / traces names.
It can be fixed to `strimzi-kafka-bridge`.

Reply:
I guess the point is, how does it differ from the environment variable and from the system property?

Reply:
I think we follow the same order -- env, sys, config.
I'm happy to drop any or all of them.

Reply:
As how OTel lookups up config -- env, sys, custom config

Reply:
So ... the bridge is using the [Vert.x Config](https://vertx.io/docs/vertx-config/java/) for getting the configuration.
As you can see [here ](https://github.com/strimzi/strimzi-kafka-bridge/blob/main/src/main/java/io/strimzi/kafka/bridge/Application.java#L95) we provides two stores for configuration: file (the application.properties) and env vars. Any env var value overrides the same parameter specified in the properties file (because how stores are added matters and in our case is `fileStore` first and then `envStore` as explained [here](https://vertx.io/docs/vertx-config/java/#_overloading_rules)).